ResearchKit/Common/ORKChoiceAnswerFormatHelper.m

```diff
@@ -170,4 +174,15 @@ - (NSArray *)selectedIndexesForAnswer:(nullable id)answer {
     
 }
 
+- (NSString *)stringForChoiceAnswer:(id)answer {
+    __block NSMutableArray<NSString *> *answerStrings = [[NSMutableArray alloc] init];
+    for (NSNumber *index in [self selectedIndexesForAnswer:answer]) {
```

Comment:
Go ahead and store `[self selectedIndexesForAnswer:answer]` instead of re-generating it for each iteration.